src/test/java/com/datalogics/pdf/samples/printing/PrintPdfTest.java

```diff
@@ -111,4 +138,55 @@ private void savePageImage(final BufferedImage image, final int pageIndex) throw
             }
         }
     }
+
+    /*
+     * TestMultiPagePrinterJob implements a 'fake' PrinterJob to intercept print requests.
```

Comment:
Since this an obscure bug, can we have some description in the comment about what this test is verifying?